llama_index/__init__.py

```diff
@@ -151,28 +88,11 @@
     # indices
     "SummaryIndex",
     "VectorStoreIndex",
-    "SimpleKeywordTableIndex",
-    "KeywordTableIndex",
-    "RAKEKeywordTableIndex",
     "TreeIndex",
-    "SQLStructStoreIndex",
-    "PandasIndex",
-    "EmptyIndex",
-    "DocumentSummaryIndex",
-    "KnowledgeGraphIndex",
     # indices - legacy names
-    "GPTKeywordTableIndex",
-    "GPTKnowledgeGraphIndex",
-    "GPTSimpleKeywordTableIndex",
-    "GPTRAKEKeywordTableIndex",
-    "GPTListIndex",
```

Comment:
Curious -- why remove some GPT prefixes and keep others?

Reply:
I would keep also GPTListIndex, others are probably fine to remove the legacy import

Reply:
yep